python/paddle/v2/framework/layers.py

```diff
@@ -729,3 +729,16 @@ def complete_rnn_op(self):
                 'states': memories,
                 'step_block': rnn_block
             })
+
+
+def lod_rank_table(x, level=0, program=None):
```

Comment:
It's better to expose a layer/op using a verb, like `rank_lod`. It's a little confusing with a noun.